Library/Formula/puddletag.rb

```diff
@@ -55,6 +49,9 @@ def install
 
     bin.install Dir[libexec/"bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    system "sh create_macos_app_bundle.sh --name Puddletag --icon puddletag.png --script #{bin}/puddletag"
```

Comment:
I changed the format of the line to your example.

Just out of curiosity: 
- Why not one line? (I am no professional programmer, but I learn everyday)
- What makes .app bundles and linkapps problematic? Okay, a shell is all one needs, but sometimes it's nice to click something for loading a GUI program.